self-hosting/prepare-system/ubuntu/prepare.sh

```diff
@@ -9,6 +9,9 @@ sudo mkdir -p /var/log/medic
 sudo apt-get update
 
 # Install Docker CE
+sudo echo \
```

Comment:
I see this as the third step when configuring the repository. Any reason to not include the curl to get the signing?

`curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg`

Reply:
Third step from here https://docs.docker.com/engine/install/ubuntu/#install-using-the-repository

Reply:
Good call. I have added more pre install steps. Kindly check and let me know.